### PR TITLE
PyCon ZA 2016

### PR DESCRIPTION
**Pull request by zoidbergwill**
Closes https://github.com/pyvideo/data/issues/84

### CODE REVIEW

File: pycon-za-2016/videos/an-introduction-to-interfacing-with-c-using-cffi.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "description": "<p>Python <a href=\"https://cffi.readthedocs.io/en/latest/\">CFFI (C Foreign Function Interface)</a> provides a powerful set of tools for interacting with C from Python.</p>\n<p>In this tutorial, I will cover the basics of using CFFI, including the various modes it can be used in. I will also cover several of the more advanced aspects, such as callbacks, and describe the mechanisms for distributing modules that use Python CFFI.</p>",
+  "duration": 4618,
+  "recorded": "2016-10-06",
+  "related_urls": [
+    "https://za.pycon.org/talks/20/"
```
Comment (stefanor) [inline pycon-za-2016/videos/an-introduction-to-interfacing-with-c-using-cffi.json]: Can you rewrite all the URLS with 2016.za.pycon.org?

File: pycon-za-2016/videos/closing.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "description": "<section class=\"section1\"><h1>Closing Ceremony</h1>\n<p>The closing ceremony. A general wrap of the conference. There will also be announcements about sprints on Saturday and Sunday.</p>\n</section>",
```
Comment (Daniel-at-github) [inline pycon-za-2016/videos/closing.json]: Description field have to be in restructured text.
This is literally what you will see if published now:
> Description
  ~~~
  <section class="section1"><h1>Closing Ceremony</h1> <p>The closing ceremony. 
  A general wrap of the conference. There will also be announcements about sprints 
  on Saturday and Sunday.</p> </section>
  ~~~

File: pycon-za-2016/videos/an-introduction-to-interfacing-with-c-using-cffi.json (hunk 1 above)
Comment (Daniel-at-github) [inline pycon-za-2016/videos/an-introduction-to-interfacing-with-c-using-cffi.json]: Done in: 3371dd1 `Replaced all the URLS to 2016.za.pycon.org`

File: pycon-za-2016/videos/closing.json (hunk 2 above)
Comment (Daniel-at-github) [inline pycon-za-2016/videos/closing.json]: Done in: d3dd762 `Converted all descriptions from html to ReStructuredText`

**Comment by Daniel-at-github** [comment]
@logston what you think about this:

Done local
~~~
...
python3 ~/git/pytube/data/.tests/shape.py -d ~/git/pytube/data -v 1
~~~

With continuous-integration/travis-ci/pr
~~~
...
python3 /home/travis/build/pyvideo/data/.tests/render_rest.py -d /home/travis/build/pyvideo/data
:11: (ERROR/3) Unexpected indentation.
:17: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
:4: (ERROR/3) Unexpected indentation.
:9: (WARNING/2) Block quote ends without a blank line; unexpected unindent.
:1: (WARNING/2) Inline emphasis start-string without end-string.
:1: (WARNING/2) Inline emphasis start-string without end-string.
make: *** [test-render-rest] Error 1
~~~
No idea what the error could be

File: pycon-za-2016/videos/closing.json (hunk 2 above)
Comment (zoidbergwill) [inline pycon-za-2016/videos/closing.json]: Thanks for fixing those issues. 🙏

**Pull request merged**